### DISCUSSION TITLE
Price Y-axis on right orientation

### DISCUSSION THREAD

**Discussion by arezi** [highfestiva/finplot#327] [Q&A]
Can I put the price on right orientation?

**Comment by highfestiva** [comment]
I've spent a few hours on trying to get that working, but no luck so far. I don't remember the details from my investigation two-three years ago, but I think it had to do with some transformation going bonkers. I'll look into again when I have time.

**Comment by highfestiva** [comment]
I got it going in 4658cc3, from now on price is always on the right. Going to make a new release as soon as I feel confident this is not causing any serious bugs.
